modules/book.js

```diff
@@ -0,0 +1,111 @@
+export default class Book {
+  constructor() {
+    this.selectElements();
+
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+
+    this.addBtn.addEventListener('click', this.addBook.bind(this));
+    this.addLink.addEventListener('click', this.addToList.bind(this));
+    this.listLink.addEventListener('click', this.displayBookList.bind(this));
+    this.listContact.addEventListener(
+      'click',
+      this.displayContactInfo.bind(this),
+    );
+
+    this.renderBooks();
+  }
+
+  selectElements() {
+    this.bookTitle = document.querySelector('#book-title');
+    this.bookAuthor = document.querySelector('#book-author');
+    this.addBtn = document.querySelector('#btn');
+    this.listEntry = document.querySelector('#list-entry');
+    this.listLink = document.querySelector('#bk-list');
+    this.addLink = document.querySelector('#bk-add');
+    this.listContact = document.querySelector('#bk-contact');
+    this.sectionList = document.querySelector('#list');
+    this.sectionNew = document.querySelector('#new');
+    this.sectionContact = document.querySelector('#contact');
+    this.showDate = document.querySelector('.show-date');
+  }
+
+  addBook() {
+    const title = this.bookTitle.value.trim();
+    const author = this.bookAuthor.value.trim();
+
+    if (title.length > 0 && author.length > 0) {
+      const book = { title, author };
+      const same = this.books.some(
+        (bk) => JSON.stringify(bk) === JSON.stringify(book),
+      );
+      if (!same) {
+        this.books.push(book);
+        this.updateStorage();
+        this.clearInputs();
+        this.renderBooks();
+      }
+    }
+  }
+
+  updateStorage() {
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
+
+  removeBook(title, author) {
+    this.books = this.books.filter(
+      (book) => book.title !== title && book.author !== author,
+    );
+    this.updateStorage();
+    this.renderBooks();
+  }
+
+  clearInputs() {
+    this.bookTitle.value = '';
+    this.bookAuthor.value = '';
+  }
+
+  renderBooks() {
+    this.listEntry.innerHTML = this.books
+      .map(
+        (book) => `
+          <li class="book-card">
+            <p class="entry">
+              <span class="book-title">"${book.title}"</span> by
+              <span class="book-author">${book.author}</span>
+            </p>
+            <button class="remove-btn">Remove</button>
+          </li>
+        `,
+      )
+      .join('');
```

Comment:
- [ ] Excellent progress in refactoring your code to leverage `ES6` syntax. Your efforts are commendable. I'd like to draw your attention to a point mentioned in the project requirements. It would be beneficial to adopt the `arrow function` syntax for your methods.

- [ ] Arrow function syntax offers a more concise way to define functions and methods, enhancing both readability and maintainability. By aligning with this syntax, your codebase will exhibit a cohesive and modern structure. It's a step towards standardizing the codebase and aligning it with best practices. See below for more details:
<img width="574" alt="Screenshot 2023-08-28 at 15 37 23" src="https://github.com/dd-obua/Mic-Awesome-Books-Modular/assets/107192076/54614c20-8799-43a7-b007-3385d779b145">


Happy coding!😊🚀🍀